### PR TITLE
add MobilityData list of initial GTFS-RT urls

### PR DESCRIPTION
**Pull request by hunterowens**
This PR starts the process of tracking GTFS RT urls. Based on a dump from Maxime at Mobility Data, this is the valid-ish URLs I could find. 

Next batches to process
 
 - TransitLand
 - GRaSS Clients

### CODE REVIEW

**Comment by thekaveman** [comment]
Incoming commit with some yaml formatting cleanup.

File: airflow/data/agencies.yml [hunk 1]
```diff
@@ -1,698 +1,752 @@
...
   itp_id: 91
 desert-roadrunner:
   agency_name: Desert Roadrunner
   gtfs_schedule_url:
-  - http://data.trilliumtransit.com/gtfs/paloverde_valley-ca-us/paloverde_valley-ca-us.zip
+    - http://data.trilliumtransit.com/gtfs/paloverde_valley-ca-us/paloverde_valley-ca-us.zip
   itp_id: 238
 dinuba-area-regional-transit-:
-  agency_name: 'Dinuba Area Regional Transit '
+  agency_name: "Dinuba Area Regional Transit "
```
Comment (thekaveman) [inline airflow/data/agencies.yml:256]: Is the `-` at the end of the agency key / space at the end of the name correct?

**Comment by hunterowens** [comment]
Nope, I think that is a processing from the spreadsheet artifact, AFAIK

File: airflow/data/agencies.yml (hunk 1 above)
Comment (thekaveman) [reply airflow/data/agencies.yml]: Fixed

**Pull request merged**